tests/contrib/gunicorn/test_gunicorn.py

```diff
@@ -187,19 +189,22 @@ def gunicorn_server(gunicorn_server_settings, tmp_path):
 )
 
 
-@pytest.mark.parametrize(
-    "gunicorn_server_settings",
-    [
-        SETTINGS_GEVENT_APPIMPORT_PATCH_POSTWORKERSERVICE,
-        SETTINGS_GEVENT_POSTWORKERIMPORT_PATCH_POSTWORKERSERVICE,
-    ],
-)
-def test_no_known_errors_occur(gunicorn_server_settings, tmp_path):
-    with gunicorn_server(gunicorn_server_settings, tmp_path) as context:
-        server_process, client = context
-        r = client.get("/")
-    assert_no_profiler_error(server_process)
-    assert_remoteconfig_started_successfully(r)
+if sys.version_info <= (3, 10):
```

Comment:
should we just use `@pytest.mark.skipif(sys.version_info > (3, 10), reason="Some note about how this doesn't apply to 3.11+ ...?")` ?